src/main/java/com/rtb/projectmanagementtool/user/AuthOps.java

```diff
@@ -0,0 +1,46 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import java.util.*;
+
+public final class AuthOps {
+
+  private DatastoreService datastore;
+  public String cookieName;
+  public String[] cookieValues;
+  public Cookie userCookie;
+
+  public AuthOps(DatastoreService datastore) {
+    this.datastore = datastore;
+    cookieName = "loginStatus";
+    cookieValues = {"out", "null"};
```

Comment:
Is this the right idea for storing the cookie vals? I researched it had to be a String so I thought a String array might work. Getting a lot of invalid expression errors.